### ISSUE TITLE
collections.Iterable not found

### ISSUE THREAD

**Issue by captcha1** [pydap/pydap#267]
When I install pydap (on Fedora 36 w/ Python 3.10.6), I need to change collections.Iterable to collections.abc.Iterable : 

```
$ diff pydap/responses/das.py-o pydap/responses/das.py
14c14
< from collections import Iterable
---
> from collections.abc import Iterable

$ diff pydap/model.py-o pydap/model.py
175c175,176
< from collections import OrderedDict, Mapping
---
> from collections import OrderedDict
> from collections.abc import Mapping
```

**Comment by headtr1ck** [comment]
Should be solved when using pydap 3.3

**Comment by Mikejmnez** [comment]
Indeed, this is solved by PR #272
